src/main/resources/org/jenkinsci/plugins/scriptsecurity/scripts/ScriptApproval/config.jelly

```diff
@@ -0,0 +1,4 @@
+<!-- Just ti satisfy GlobalConfiguration.getGlobalConfigPage() -->
```

Comment:
```suggestion
<!-- Just to satisfy GlobalConfiguration.getGlobalConfigPage() -->
```

Reply:
nit: Looks like the typo is still there.